students/10th/songsuho/main.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <nav>
+      <div class="logo">
+        <div class="logo_mini"><img src="img/logo.png" alt="인스타그램 로고" width="30px" height="30px"></div>
+        <div class="logo_bar"><img src="img/westagram.png" alt="인스타그램 글자" width="150px" height="50px"></div>
+      </div>
+      <input class="point" type="text" placeholder="검색">
+      </input>
+      <div class="menu">
+        <div class="compass"><img src="img/explore.png" alt="탐색" width="30px" height="30px"></div>
+        <div class="heart"><img src="img/heart.png" alt="하트" width="30px" height="30px"></div>
+        <div class="profile"><img src="img/profile.png" alt="프로필" width="30px" height="30px"></div>
```

Comment:
```suggestion
        <div class="compass">
          <img src="img/explore.png" alt="탐색" width="30px" height="30px">
        </div>
        <div class="heart">
          <img src="img/heart.png" alt="하트" width="30px" height="30px">
        </div>
        <div class="profile">
          <img src="img/profile.png" alt="프로필" width="30px" height="30px">
        </div>
```

크게 상관은 없을 거 같은데 div 태그 안에 img 태그가 포함되어 있다는 점을 좀 더 명확히 하기 위해서 위 처럼 바꿔 쓰는걸 추천드립니다!

그리고 여기도 inline 스타일... css로 옮겨주세요!

Reply:
수정하였습니다.